http-server/src/test/java/org/triplea/server/lobby/chat/event/processing/ChatEventProcessorTest.java

```diff
@@ -72,29 +72,30 @@ void connect() {
           responses.get(0),
           is(
               ServerResponse.backToClient(
-                  ServerMessageEnvelopeFactory.newPlayerListing(List.of(CHAT_PARTICIPANT_0)))));
+                  ChatServerEnvelopeFactory.newPlayerListing(List.of(CHAT_PARTICIPANT_0)))));
```

Comment:
Note, this is just a rename "ServerMessageEnvelopeFactory" -> "ChatServerEnvelopeFactory". We have different kinds of messages now.